docs/combine-active-promotions.md

```diff
@@ -0,0 +1,33 @@
+# Combine Active Promotions
+
+VirtoCommerce Marketing Module represents two basic promotion combination policies:  
+
+1. **Best reward policy** - select the promotion reward, that most of all promotions benefits the customer.  
+1. **Combine stackable policy** - If your store has multiple active promotions, shoppers can qualify for multiple promotions at a time.  
+
+## Combine Stackable Policy
+
+With this policy each promotion can be combined with all other promotions in an order. In addition, resulting promotion rewards can be stacked on the top of other promotions rewards in the same reward group (shipment, item, order), which overrides certain promotion policies. The priority of promotions determines whether a stackable promotion reward applies to the order. 
+
+The following table detail how promotions rewards are applied compared to other promotions. How promotions are applied is based on different priorities and the exclusivity with other promotions settings:
+
+![Fig. Table](media/screen-table.png)
+
+**Important**: Stackable combination policy has a constraint- if the total order or any object price is less than zero after a promotion is applied. If the total order or entry price is less than zero, then this reward will be skipped and taken next reward by promotion by priority.
```

Comment:
Maybe better to rephrase:
Stackable combination policy has a constraint - if the total order or any object price is less than zero after a promotion is applied, then this reward will be skipped and next reward by priority will be taken.